src/screens/OrgAdminHomePage/OrgAdminHomePage.tsx

```diff
@@ -11,15 +11,33 @@ import {
   faGooglePlus,
   faLinkedinIn,
 } from '@fortawesome/free-brands-svg-icons';
+import { useQuery } from '@apollo/client';
+import { ORGANIZATIONS_LIST } from 'GraphQl/Queries/Queries';
 
 function OrgAdminHomePage(): JSX.Element {
+  const a = window.location.href.split('=')[1];
+
+  const b = '/orgmember/id=' + a;
```

Comment:
"a" and "b" are **not** descriptive variable names.

Please make use of descriptive names for your variable declarations